docs/quickstart.md

```diff
@@ -0,0 +1,760 @@
+    Globals:
+        Function:
+            Timeout: 3
+    Resources:
+        HelloWorldFunction:
+            Type: AWS::Serverless::Function
+            Properties:
+            CodeUri: hello_world/
+            Handler: app.lambda_handler
+            Runtime: python3.9
+            Events:
+                HelloWorld:
+                Type: Api 
+                Properties:
+                    Path: /hello
+                    Method: get
+        HelloWorldFunctionName:
+            Type: AWS::Serverless::Function
+            Properties:
+            CodeUri: hello_world/
+            Handler: app_name.lambda_handler
+            Runtime: python3.9
+            Events:
+                HelloWorldName:
+                Type: Api
+                Properties:
+                    Path: /hello/{name}
+                    Method: get
+    Outputs:
+        HelloWorldApi:
+            Description: "API Gateway endpoint URL for Prod stage for Hello World function"
+            Value: !Sub "https://${ServerlessRestApi}.execute-api.${AWS::Region}.amazonaws.com/Prod/hello/"
+    ```
+
+This way definitively works. The downside of this approach is that you need to create new lambda file for every method or path and also configure SAM template to add new lambda configuration and configure API Gateway to point into them. For small additional methods it might be too much boilerplate code.  
+Another approach though that might be more suitable in this case is to have all of our methods under control of one lambda and "route" into specific method based on a request. 
+!!! Info
+    If you are interested in more detailed explanation between those two approaches we explained considerations [here](../core/event_handler/api_gateway/#considerations)
+Example, naive code might look like the following code snippet.
+=== "app.py"
+
+    ```python hl_lines="10-23 26-28 34-35"
+        def hello_name(event, **kargs):
+            username = event["pathParameters"]["name"]
+            return {"statusCode": 200, "body": {"message": f"hello {username}!"}}
+
+
+        def hello(**kargs):
+            return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+
+        class Router:
+            def __init__(self):
+                self.routes = {}
+
+            def set(self, path, method, handler):
+                self.routes[f"{path}-{method}"] = handler
+
+            def get(self, path, method):
+                try:
+                    route = self.routes[f"{path}-{method}"]
+                except:
+                    print("Cannot route request to correct method")
+                    raise NotImplemented
+                return route
+
+
+        router = Router()
+        router.set(path="/hello", method="GET", handler=hello)
+        router.set(path="/hello/{name}", method="GET", handler=hello_name)
+
+
+        def lambda_handler(event, context):
+            path = event["resource"]
+            http_method = event["httpMethod"]
```

Comment:
no error handling here, not sure if intentional, if yes, just ignore my comment.

Reply:
this is on purpose to keep it simple